src/ui/inspectorPanel/inspectorPanel.ts

```diff
@@ -24,6 +25,7 @@ export const styles =
     overflow-y: auto;
     z-index: calc(var(--z-index) + 10);
   }
+
```

Comment:
No newlines between `.foo` and simple pseudo class selecting the same element `.foo:bar`, for consistency.